src/sdam/description/topology/test/sdam.rs

```diff
@@ -76,7 +145,15 @@ async fn run_test(test_file: TestFile) {
         .expect(&test_file.description);
 
     let test_description = &test_file.description;
-    let mut topology_description = TopologyDescription::new(options).expect(test_description);
+
+    // TODO: RUST-360 unskip tests that rely on topology version
+    if test_description.contains("topologyVersion") {
+        println!("Skipping {} (RUST-360)", test_description);
+        return;
+    }
+
+    let topology = Topology::new_mocked(options.clone());
```

Comment:
Because we need to handle errors as part of this runner now, we need to go through the `Topology` instead of the `TopologyDescription`, which required some changes to the `Topology` to support "mocked" scenarios (i.e. don't start any monitors for new servers).